doc/Customization-guide/How-to-contribute/Documentation-contribution-and-style-guide/index.md

```diff
@@ -11,62 +11,168 @@ sort_idx: 30
+Questions can work well when users actually do have them, not when we invent them on their behalf.
+* Use exclamation points sparingly.
+The overuse of exclamation points can make content seem effusive or ingratiating.
+Save exclamation points for when they count.
+* Be mindful of globalization and cultural considerations before you use colloquialisms or idioms.
+* Do not use non-English words or phrases, such as _de facto_ or _ad hoc_, in English content, even if you think these terms are generally known and understood.
+
+## Use of terminology
+* Whenever possible, you should get your point across by using common English words.
+It is all right to use technical terms when they are necessary for precise communication, even with a general audience, but do not write as if everybody understands these terms or will immediately grasp their meaning.
+* Define terms in the text as you introduce them.
+* Spell out an acronym on its first mention on the page, for example, _Common Type Library (CTL)_.
+* Provide links from the main text to the glossary.
+* Do not create a new term if a term describing a concept already exists in the glossary.
+* If you need to create a new term, verify that the term that you select is not already in use to mean something else.
+
+
+## Grammar
+* Give preference to present tense over past and future tenses.
+* Give preference to simple verb forms over perfect and continuous forms.
+* Put the action of the sentence in the verb, not in the nouns.
+* Choose single-word verbs over multiple-word verbs.
+* Don’t convert verbs to nouns and nouns to verbs.
+* Do not use a transitive verb without a direct object.
+Either supply a direct object or use an intransitive verb instead.
+* Avoid using weak, vague verbs such as _be_, _have_, _make_, and _do_.
+Such verbs are sometimes necessary, but use a more descriptive verb whenever you can.
+* Use imperative mood for procedures, formulate your instructions as giving commands to the user.
+* Use indicative mood for conveying information such as facts, questions, assertions, or explanations.
+* Omit unnecessary adverbs.
+* Use context to reduce ambiguity.
+Some words can be read as both verbs and nouns, such as _file_, _post_, _input_, etc.
+When using these terms, ensure that the context and sentence structure reduce ambiguity.
+* In general, use second person (pronoun _you_) also known as _direct address_.
+Second person supports a friendly tone because it connects you with the user.
+* Use first person sparingly.
+First person is appropriate when writing from the point of view of the user.
+* Avoid using more than 3 nouns in a noun string as it may be confusing or ambiguous even for native English speakers.
+
+## Text formatting
+* Use **bold** for:
+  * Titles of windows and dialog boxes.
+  * Names of commands, attributes, constants, methods, fields, predefined classes, databases, events, UI elements.
+  * User input.
+* Use _italic_ for:
+  * Technical terms on first mention.
```

Comment:
I tend to agree that there is too much bold. I would reduce the amount of bold stuff in favor of term definitions. These are important. There is no good reason, for example, why user input should be bold.
Would I open a page to quickly find user input? No.
Would I open a page to quickly find key terms? Yes.